project.yaml

```diff
@@ -41,11 +80,13 @@ actions:
   ## Add ethnicity
   join_cohorts:
     run: >
-      cohort-joiner:v0.0.46
+      cohort-joiner:v0.0.48
         --lhs output/input_*.csv
         --rhs output/input_ethnicity.csv
         --output-dir output/data
-    needs: [generate_cohort, generate_ethnicity_cohort]
+    needs: [generate_study_population_1, generate_study_population_2, 
+      generate_study_population_3, generate_study_population_4, 
+      generate_study_population_5, generate_ethnicity_cohort]
```

Comment:
Just for the initial run please could you remove the unused `generate_study_population_i`  here as discussed.

Reply:
Done